sdk/iot/hub/samples/src/paho_iot_hub_methods_example.c

```diff
@@ -23,21 +23,21 @@
 //              Note: this is required to work-around MQTTClient.h as well as az_span init issues.
 #include <_az_cfg.h>
 
-// Device ID
-#define DEVICE_ID "AZ_IOT_DEVICE_ID"
+// DO NOT MODIFY: Device ID Environment Variable Name
+#define ENV_DEVICE_ID "AZ_IOT_DEVICE_ID"
 
-// IoT Hub Hostname
-#define IOT_HUB_HOSTNAME "AZ_IOT_HUB_HOSTNAME"
+// DO NOT MODIFY: IoT Hub Hostname Environment Variable Name
+#define ENV_IOT_HUB_HOSTNAME "AZ_IOT_HUB_HOSTNAME"
 
-// AZ_IOT_DEVICE_X509_CERT_PEM_FILE is the path to a PEM file containing the device certificate and
+// DO NOT MODIFY: The path to a PEM file containing the device certificate and
 // key as well as any intermediate certificates chaining to an uploaded group certificate.
-#define DEVICE_X509_CERT_PEM_FILE "AZ_IOT_DEVICE_X509_CERT_PEM_FILE"
+#define ENV_DEVICE_X509_CERT_PEM_FILE "AZ_IOT_DEVICE_X509_CERT_PEM_FILE"
 
-// AZ_IOT_DEVICE_X509_TRUST_PEM_FILE is the path to a PEM file containing the server trusted CA
+// DO NOT MODIFY: the path to a PEM file containing the server trusted CA
```

Comment:
Same nit here/elsewhere.
```suggestion
// DO NOT MODIFY: The path to a PEM file containing the server trusted CA.
```